client/src/Components/Pages/Questionnaire/PersonalDetails1/index.js

```diff
@@ -1,65 +1,158 @@
+            const currentRejectFileSize = currentRejectFile.size;
+            
+            
+            if ( currentRejectFileSize > fileMaxSize) {
+                alert('This file is too big..please choose another one')
+            }
+        }
+            const currentFile = files[0];
+            const currentFileSize = currentFile.size;
+            if ( currentFileSize > fileMaxSize) {
+                alert('This file is too big..please choose another one')
+            
+            }else{
+            this.setState({files:files,label:files[0].name})
+        }
+    }
       
     render() {
-        const { selectedOption } = this.state; 
         return ( 
-            <div>
-                <div className="third-section">
-                    <h2>Now we’ve got the boring stuff out of the way, let’s get to know you better</h2>
-                    <div className="six-card">
-                        <p>What subjects are you studying? Tick all that apply</p>
-                        <Select
-                            value={selectedOption}
-                            isMulti={this.handleChange}
-                            options={options}
-                        />
-                        <p>What are your favourite subjects Azara ? Tick up to 3</p>
-                        <input type="text" className ="sec1-input"/>
-                        <p>What else do you enjoy at school? Tell us as much as you can.</p>
-                        <textarea rows="5" cols="40" placeholder="Answer"></textarea>
-                        <p>What are your interests and hobbies outside of school? Tell us everything you enjoy doing.</p>
-                        <textarea rows="5" cols="40" placeholder="Answer"></textarea>
-                    </div>
-                    <div className="seventh-card">
-                        <p>If you have a copy of your CV, please upload it to your dashboard</p>
-                        <input type="file"/>
-                        <p>Can you imagine your future career? What does it look like?</p>
-                        <textarea rows="5" cols="40" placeholder="Answer"></textarea>
-                        <p>What kinds of jobs do you think will interest you?</p>
-                        <textarea rows="5" cols="40" placeholder="Answer"></textarea>
+            <div className="container">
+                <div>
+                    <h2>Now we've got the boring stuff out of the way, let's get to know more about your school studies</h2>
+                    <div className="third-section">  
+                        <div className="six-card">
+                            <span className="num" id="num6">6</span>
+                            <p>What subjects are you studying? Tick all that apply</p>
+                            <Select
+                                className="select-input"
+                                onChange={this.handleSubjectChange}
+                                value={this.subjects}
```

Comment:
I think you need to change this to `value={this.state.subjects}`.  This is similar to what I was saying in the other pull request - your change handler is calling `this.setState({subjects})`, and `this.setState` stores data in `this.state`.
For example, if you called
`this.setState({ firstName: 'Tom' })`,
then the result is that `this.state.firstName` is Tom.  setState puts the data into this.state.  So when your change handler calls `this.setState({subjects})`, you're putting that data into `this.state.subjects`.  Therefore, you want this `<Select>` component's value prop to use `this.state.subjects`, in order for it to properly display the subjects that the user has just picked.  Otherwise, the select component might not show the correct items as the user selects different items.

Reply:
Actually, it worked even without put state but yes you are right I edit it #95